tests/src/test/java/alluxio/server/worker/WorkerFuseIntegrationTest.java

```diff
@@ -33,7 +33,13 @@ public void mountFuse(FileSystem fileSystem, String mountPoint, String alluxioRo
   }
 
   @Override
-  public void umountFuse(String mountPath) throws Exception {
+  public void beforeStop() throws Exception {
     // Fuse application is unmounted automatically when stopping the worker
   }
+
+  @Override
+  public void afterStop() throws Exception {
+    // umount the mountpoint
+    umountFromShellIfMounted();
```

Comment:
worker fuse should be unmounted directly when stopping the worker process.
this line can be added to double check if fuse is unmounted but is not necessary